pkgs/servers/ldap/lldap/default.nix

```diff
@@ -44,7 +44,8 @@ let
 
     patches = [
       (fetchpatch {
-        url = "https://github.com/lldap/lldap/pull/947.patch";
+        ## https://github.com/lldap/lldap/pull/947
+        url = "https://github.com/lldap/lldap/compare/6aa9303339ac1793b4be9ae01c56a6369a5d77f0...620939be9c8c32f8ace4519cc49a7ac07d88fe87.patch";
```

Comment:
This is also not a safe URL to use — if I have to e.g. rebase the PR, that commit will get deleted when GitHub GCs.

Reply:
(But if you want to just wait a bit and see if we get an upstream release, that's totally fine — we have plenty of time.)

Reply:
Have you actually verified that behavior? Because from all I know, github will retain all commits it ever saw: https://trufflesecurity.com/blog/anyone-can-access-deleted-and-private-repo-data-github

Reply:
Yes, I have.  You can see it if you go to any older PR and try to look at previous versions of it prior to a force push. They don't do it immediately, but every so often, so that post presumably relies on finding data they haven't deleted yet.

Reply:
That's weird. In a story about that, I specifically remember a GH spokesperson being cited, who said that they might look at restricting what they retain in the future, but do not at this point.

In any case, I've pushed a patch file.

Reply:
Well, it's true that they don't "restrict" anything — they just garbage collect from time to time, which isn't any sort of permissions system.